<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWidget</class>
 <widget class="QWidget" name="SettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="text">
       <string>IP设置</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="IP_1"/>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="IP_2"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="IP_3"/>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="IP_4"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>端口配置</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="port"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="finish">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>290</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>完成</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>显示可行落子位置</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>服务端端口配置</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="port_server"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>finish</sender>
   <signal>clicked()</signal>
   <receiver>SettingsWidget</receiver>
   <slot>saveSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>302</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>saveSettings()</slot>
 </slots>
</ui>
